rustfmt.toml

```diff
@@ -0,0 +1,5 @@
+force_explicit_abi = true
+max_width = 120
```

Comment:
Can we get rid of this line? I think `100` is fine, and our original motivation for 120 (do what fuchsia does) is less relevant now.